capstone/home.html

```diff
@@ -0,0 +1,134 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="css/style.css" />
+  <link href="https://fonts.googleapis.com/css2?family=Lato&display=swap" rel="stylesheet">
+  <title>Rock & Roll Hall of Fame</title>
+</head>
+<body>
+  <header>
+    <div class="topBar">
+      <i class="fa-brands fa-facebook-f"></i>
+      <i class="fa-brands fa-twitter"></i>
+      <a href="#">English</a>
+      <a href="#">My page</a>
+      <a href="#">Logout</a>
+    </div>
+    <nav>
+    <div class="navbar">
+      <a class="logo" href="home.html" ><img src="images/logo.svg"></a>
+      <div class="nav-items">
+        <a class="aboutPage" href="about.html">About</a>
+        <a href="home.html#program">Program</a>
+        <a href="home.html#join">Join</a>
+        <a href="#sponsor">Sponsor</a>
+        <a href="about.html#news">News</a>
+        <a class="rockHall" href="#rockhall">Rock Hall</a>
+      </div>
+    </div>
+    </nav>
+    <div class="menu">
+      <a class="menu-button open" href="#"><i class="fa-solid fa-bars"></i></a>
+      <a class="menu-button close" href="#"><i class="fa-solid fa-xmark"></i></a>
```

Comment:
- [ ] You did a great job 😊 at the functionality of the hamburger icon, but when I click on it the style of the button has become like that and changes, so please try to fix the color when I visit or click the button to be like the Behance 😃

> Check below to see the difference 😊⤵️
> If you can not see the image clearly just click on it to open a new tab in the browser😊

| Figma design | Your design| 
| --- | --- |
|  ![image](https://user-images.githubusercontent.com/83360636/217937286-3ac8c1dc-f891-4c42-8bfc-6e6b6942fdf5.png)|    ![image](https://user-images.githubusercontent.com/83360636/217937254-4bbac86c-d1f2-49e0-a8bb-a1c015cf4d49.png)  |